### ISSUE TITLE
Open requests dont appear in section when log in as a Volunteer

### ISSUE THREAD

**Issue by PhilArsHermit** [reach4help/reach4help#643]
**Describe the bug**
When a "Person in needs" create a request nearby, the "Volunteer" will see it on the map, but it will not appear if he checks from the "Open Requests" tab

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'https://reach4help-staging-web-client.firebaseapp.com/requests/find' with 2 different user; one Volunteer and one Person in Needs
2. Have the Person in Needs create a request
3. After it's done, have the Volunteer refresh his page
4. The Volunteer will see the request on the Map
![Volunteer_Request_1](https://user-images.githubusercontent.com/65713824/83890472-d18e3400-a719-11ea-8894-2220646af6c7.png)
5. Open the "Open Requests" section
![Volunteer_Request_2](https://user-images.githubusercontent.com/65713824/83890493-d94dd880-a719-11ea-93ec-4e2a7c469057.png)

**Expected behavior**
The Volunteer will see 0 request, even though he was able to see it on the map

**Desktop (please complete the following information):**
Build 1
 - OS: Windows 10 Pro
 - Browser Chrome
 - Version  83.0.4103.97

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/643)

*This information is updated automatically. To modify it, please use ZenHub.*

**Comment by ashwinkjoseph** [comment]
so according to the UX that was explained, the open requests to a CAV would be those requests to which the CAV has responded to by making offers but the PIN hasn't yet responded by accepting the offers.

**Comment by schowdhury8** [comment]
As @ashwinkjoseph, "Open Requests" aren't ones available near you, they're ones that you offered help for but you haven't been accepted/rejected yet. As of now, we've implemented a list-view of nearby requests similar to what this issue is referring to. 
![image](https://user-images.githubusercontent.com/29135917/86046109-77f5fc80-ba6e-11ea-95fc-b54f2ec469db.png)
Closing for now, but reopen if necessary.
